tasks/publish-to-cgw/publish-to-cgw.yaml

```diff
@@ -183,11 +183,11 @@ spec:
             yaml.dump(metadata, file, default_flow_style=False, sort_keys=False)
 
         print(f"YAML content dumped to {METADATA_FILE_PATH}")
+
         command = [
             'push-cgw-metadata',
```

Comment:
what provides this again?

Reply:
It's placed in the environment around line 50 of this file. Then the `push-cgw-metadata` python script picks it up here: https://github.com/release-engineering/pubtools-content-gateway/blob/main/src/pubtools/_content_gateway/push_cgw.py#L88

Reply:
sorry:  https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L49